components/InfoBox/index.jsx

```diff
@@ -92,11 +95,11 @@ export default class InfoBox extends React.Component {
 
   render () {
     return (
-      <p
+      <div
         ref={container => { this.container = container; }}
         className={`${styles.infoBox} ${styles[this.props.position]}`}>
         {this.props.content}
-      </p>
+      </div>
```

Comment:
Since it would be nice, to still have a p-tag, if the tooltip is only a single line of text, it would be nice, if you could check, if content is a simple string and only if it is an object, switch to a div tag